cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -91,7 +91,12 @@ use bridge_hub_common::{
 use pallet_xcm::EnsureXcm;
 pub use sp_consensus_aura::sr25519::AuthorityId as AuraId;
 pub use sp_runtime::{MultiAddress, Perbill, Permill};
-use xcm::VersionedLocation;
+
+#[allow(unused_imports)]
+use xcm::{
+	latest::{ROCOCO_GENESIS_HASH, WESTEND_GENESIS_HASH},
+	VersionedLocation,
+};
```

Comment:
Ok, so they're only used in `runtime-benchmarks`, you can import them conditionally like this:

```suggestion
#[cfg(feature = "runtime-benchmarks")]
use xcm::latest::{ROCOCO_GENESIS_HASH, WESTEND_GENESIS_HASH};
use xcm::VersionedLocation;
```